docs/next/guides/integrate-with-angular/angular-modules.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Using modules with Angular
+metaTitle: Using modules with Angular - Guide - Handsontable Documentation
+permalink: /next/angular-modules
+canonicalUrl: /angular-modules
+---
+
+# Using modules with Angular
+
+[[toc]]
+
+## Overview
+
+To reduce the size of your app, you can use Handsontable by importing individual [modules](@/guides/building-and-testing/modules.md).
+
+## Using modules with Angular
+
+To use modules with Handsontable's [Angular wrapper](@/guides/integrate-with-angular/angular-installation.md), follow the steps below:
+
+### Step 1: Import the `Base` module
```

Comment:
```suggestion
### Step 1: Import the `handsontable/base` module
```
I think that the `Base` name here can be too generic for new developers. What do you think about it to call it the same as the package is named?